src/module_2/module_2_EDA.md

```diff
@@ -0,0 +1,2197 @@
+    Número total de valores: 92361
+    Número de valores nulos: 0
+    Número de valores únicos: 7
+    Frecuencias de los valores únicos:
+    count_children
+    0    88196
+    2     1973
+    1     1297
+    3      801
+    4       69
+    5       14
+    6       11
+    Name: count, dtype: int64
+    
+    --------------------------------------------------
+    
+    Análisis de la columna: count_babies
+    Tipo: Numérica
+    Media: 0.005933240220439363
+    Número total de valores: 92361
+    Número de valores nulos: 0
+    Número de valores únicos: 3
+    Frecuencias de los valores únicos:
+    count_babies
+    0    91905
+    1      364
+    2       92
+    Name: count, dtype: int64
+    
+    --------------------------------------------------
+    
+    Análisis de la columna: count_pets
+    Tipo: Numérica
+    Media: 0.06237481187947294
+    Número total de valores: 92361
+    Número de valores nulos: 0
+    Número de valores únicos: 7
+    Frecuencias de los valores únicos:
+    count_pets
+    0    88634
+    1     2492
+    2      700
+    3      344
+    4      140
+    5       29
+    6       22
+    Name: count, dtype: int64
+    
+    --------------------------------------------------
+    
+    Análisis de la columna: number_of_orders
+    Tipo: Numérica
+    Media: 4.075356481631857
+    Número total de valores: 92361
+    Número de valores nulos: 0
+    Número de valores únicos: 23
+    Frecuencias de los valores únicos:
+    number_of_orders
+    1     35273
+    2     14554
+    3      8097
+    4      6388
+    5      5168
+    6      4041
+    7      3979
+    8      2687
+    9      2375
+    14     1950
+    10     1653
+    11     1287
+    13     1089
+    21      964
+    12      733
+    22      478
+    16      382
+    17      347
+    18      278
+    19      245
+    25      167
+    15      139
+    20       87
+    Name: count, dtype: int64
+    
+    --------------------------------------------------
+    
+    Análisis de la columna: variant_id_count
+    Tipo: Numérica
+    Media: 6.696744296835244
+    Número total de valores: 92361
+    Número de valores nulos: 0
+    Número de valores únicos: 67
+    Frecuencias de los valores únicos:
+    variant_id_count
+    0      47218
+    1      10017
+    2       4231
+    3       2985
+    5       2283
+           ...  
+    52        26
+    73        25
+    51        19
+    45        16
+    320        9
+    Name: count, Length: 67, dtype: int64
+    
+    --------------------------------------------------
+    
+
+
+## Insights
+
+Popularidad de producto:
+- Popularidad de producto por marca: Al pintar a los 20 vendors que tienen las variantes de producto más populares nos dimos cuenta de que marcas "bio" como oatly y plenty ocupaban las primeras posiciones por lo que es posible o bien que la tienda sea un herbolario donde todas las marcas son de esta tipología o que simplemente este tipo de marcas sean las preferidas por los consumidores.
```

Comment:
It is very interesting to visualize the `product_types` of the most popular products (instead of the brands), I encourage you to do it.

Reply:
`que marcas "bio" como oatly y plenty ocupaban las primeras posiciones`-> this is more related to the `product_type` of these brands, not the "herb" nature of them